From 7e5571a2c58313b995003d787e527f792dea3c62 Mon Sep 17 00:00:00 2001
From: Francois Romieu <romieu@fr.zoreil.com>
Date: Mon, 18 Dec 2006 00:00:55 +0100
Subject: [PATCH] r8169: remove extraneous Cmd{Tx/Rx}Enb write
Content-Length: 833
Lines: 25

Checked in Realtek's driver, this one has no business being there.
The driver works with the extra writre but there is a very noticeable
performance drop.

The patch fixes http://bugzilla.kernel.org/show_bug.cgi?id=7627

Signed-off-by: Francois Romieu <romieu@fr.zoreil.com>
---
 drivers/net/r8169.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/net/r8169.c b/drivers/net/r8169.c
index 1489100..577babd 100644
--- a/drivers/net/r8169.c
+++ b/drivers/net/r8169.c
@@ -1882,7 +1882,6 @@ static void rtl8169_hw_start(struct net_
 	    (tp->mac_version == RTL_GIGA_MAC_VER_02) ||
 	    (tp->mac_version == RTL_GIGA_MAC_VER_03) ||
 	    (tp->mac_version == RTL_GIGA_MAC_VER_04))
-		RTL_W8(ChipCmd, CmdTxEnb | CmdRxEnb);
 		rtl8169_set_rx_tx_config_registers(tp);
 
 	cmd = RTL_R16(CPlusCmd);
-- 
1.4.2.3

